packages/desktop/src/hooks/server-connection.js

```diff
@@ -1,8 +1,16 @@
 import React from "react";
 import { API_ENDPOINT } from "../constants/api";
+import { identity } from "../utils/function";
 
 const clientEventMap = {
```

Comment:
This is a bit involved. The purpose is to decouple the server stuff and Pusher from the client as much as possible. Any changes to format or names of events on the backend should have minimal impact on the client code.